packages/eth2.0-spec-test-util/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@chainsafe/eth2.0-spec-test-util",
+  "version": "0.4.0",
+  "description": "Spec test suite generator from yaml test files",
+  "author": "ChainSafe Systems",
+  "license": "LGPL-3.0",
```

Comment:
can we keep it Apache 2.0?

Reply:
@GregTheGreek ^^

Reply:
Yes - it's fine to make this apache2.0, Lodestar and the lighclient should remain LGPL3.0